src/services/crypto.service.ts

```diff
@@ -452,6 +452,12 @@ export class CryptoService implements CryptoServiceAbstraction {
         return b64DecValue;
     }
 
+    sha1(password: string) {
```

Comment:
I'd rather use web crypto for this to further reduce our dependency on Forge. Web crypto will turn this into a promise-based return. https://github.com/diafygi/webcrypto-examples#sha-1---digest